### PR TITLE
{tools}[foss/2023a] nanoQC v0.9.4

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/dc5d1905abd3347e0e61bbf0cf72eaac for a full test report.

File: easybuild/easyconfigs/n/nanoQC/nanoQC-0.9.4-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+download_dep_fail = True
+use_pip = True
+
+options = {'modulename': 'nanoQC'}
+
+sanity_check_commands = ["nanoQC --help"]
+
+sanity_pip_check = True
+
+moduleclass = 'tools'
```
Comment (verdurin) [inline easybuild/easyconfigs/n/nanoQC/nanoQC-0.9.4-foss-2023a.eb:31]: Shouldn't this be `bio`?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/n/nanoQC/nanoQC-0.9.4-foss-2023a.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/n/nanoQC/nanoQC-0.9.4-foss-2023a.eb]: Thanks.

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**